e2e/app/component/runtime-panel.ts

```diff
@@ -0,0 +1,192 @@
+import Container from 'app/container';
+import {LinkText} from 'app/text-labels';
+import Button from 'app/element/button';
+import {Page} from 'puppeteer';
+import {waitForAttributeEquality, waitWhileLoading} from 'utils/waits-utils';
+import PrimereactInputNumber from 'app/element/primereact-input-number';
+import SelectMenu from "./select-menu";
+import {savePageToFile, takeScreenshot} from "../../utils/save-file-utils";
+import BaseElement from "../element/base-element";
+
+const defaultXpath = '//*[@id="runtime-panel"]';
+const startStopIconXpath = '//*[@data-test-id="runtime-status-icon"]';
+
+export enum StartStopIconState {
+  Error = 'error',
+  None = 'none',
+  Running = 'running',
+  Starting = 'starting',
+  Stopped = 'stopped',
+  Stopping = 'stopping'
+}
+
+export enum ComputeType {
+  Standard = 'Standard VM',
+  Dataproc = 'Dataproc Cluster'
+}
+
+export enum RuntimePreset {
+  GeneralAnalysis = 'General Analysis',
+  HailGenomicsAnalysis = 'Hail Genomics Analysis'
+}
+
+export default class RuntimePanel extends Container {
+  constructor(page: Page, xpath: string = defaultXpath) {
+    super(page, xpath);
+  }
+
+  async clickCreateButton(): Promise<void> {
+    const button = await Button.findByName(this.page, {name: LinkText.Create}, this);
+    await button.click();
+    return await page.waitForTimeout(2000);
+  }
+
+  async clickCustomizeButton(): Promise<void> {
+    const button = await Button.findByName(this.page, {name: LinkText.Customize}, this);
+    await button.click();
+    return await page.waitForTimeout(2000);
```

Comment:
why do some of these click actions have a timeout while others do not? Even if click triggers an action that takes a few seconds to complete, I think it's OK to return once the click has completed. We can let the calling function add a waitForVisible or w/e else is appropriate to wait for the action to complete.

This is mainly b/c of the semantics of a function named "click". If the function was just named "customizeButton", I think it makes sense to add the wait call here.

Reply:
Oops, I think this one is because I wanted it to pause when running the test on actual (vs headless) chromium so I could see what it's doing. I'll scan through and remove any of these that are unnecessary, and comment on the ones that are